tools-image/Dockerfile

```diff
@@ -3,84 +3,59 @@ ARG LEAP_VERSION=15.4
 ARG LUET_VERSION=0.33.0
 FROM quay.io/luet/base:$LUET_VERSION AS luet
 
-FROM opensuse/leap:$LEAP_VERSION as luet-install
+FROM golang:1.20 as enki
+ENV CGO_ENABLED=0
+COPY ./enki /src/enki
```

Comment:
This guy? https://diesel-ai-art.com/cdn/shop/products/enki-hd-download-digital-405.png?v=1680768259

Reply:
indeed!